fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/storage/base/FilteringStorage.java

```diff
@@ -220,5 +220,10 @@ public long getAmount() {
 		public long getCapacity() {
 			return backingView.getCapacity();
 		}
+
+		@Override
+		public StorageView<T> getUnderlyingView() {
+			return backingView;
```

Comment:
Shouldn't this be calling `backingView.getUnderlyingView()` for preventing doubly filtering storage?

Reply:
Indeed! Fixed.